src/posh-git.psd1

```diff
@@ -4,7 +4,7 @@
 RootModule = 'posh-git.psm1'
 
 # Version number of this module.
-ModuleVersion = '1.0.0.0'
+ModuleVersion = '1.0.0.1'
```

Comment:
I take it you already have a built version of 1.0.0 set aside to publish to the gallery?

Reply:
> I take it you already have a built version of 1.0.0 set aside to publish to the gallery?

![image](https://user-images.githubusercontent.com/133987/110739681-45082900-81f7-11eb-9409-6681ffb30d8f.png)

Reply:
The 3rd and 4th downloads are mine. :-)